cc.spek.Spek.metainfo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>cc.spek.Spek</id>
+    <launchable type="desktop-id">cc.spek.Spek.desktop</launchable>
+    <name>Spek</name>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>GPL-3.0-only</project_license>
+    <summary>Acoustic Spectrum Analyser</summary>
+    <description>
+        <p>Spek is an acoustic spectrum analyser written in C++. It uses FFmpeg libraries for audio decoding and wxWidgets for the GUI.</p>
+    </description>
+    <developer id="www.spek.cc">
+        <name>Alexander Kojevnikov</name>
+    </developer>
+    <update_contact>joshua.gage.stone@gmail.com</update_contact>
+    <url type="homepage">https://www.spek.cc/</url>
+    <url type="bugtracker">https://github.com/alexkay/spek/issues</url>
+    <url type="translate">https://explore.transifex.com/spek/spek/</url>
+    <screenshots>
+        <screenshot type="default">
+            <image>https://substack-post-media.s3.amazonaws.com/public/images/87fd2840-4ee7-492e-b15f-38a384a56a4c_724x590.png</image>
+        </screenshot>
+    </screenshots>
+    <content_rating type="oars-1.1">
```

Comment:
1.1 means you don't need the `none` values just put `<content_rating type="oars-1.1" />`